recipes/mageck/meta.yaml

```diff
@@ -1,14 +1,14 @@
 package:
   name: mageck
-  version: "0.5.7"
+  version: "0.5.7a"
 
 source:
-  fn: liulab-mageck-a8c3ba10c00d.tar.gz
-  url: https://bitbucket.org/liulab/mageck/get/v0.5.7.tar.gz
-  md5: 55db9f702e6c49e7a680c967992350a0
+  fn: liulab-mageck-af89046a812b.tar.gz
+  url: https://bitbucket.org/liulab/mageck/get/v0.5.7a.tar.gz
+  md5: 3ab8665f32da9bb4293a6f1175c9b8a4
 
 build:
-  number: 2
+  number: 3
```

Comment:
this should be 0 if you increase the version number

Reply:
Thank you for pointing this out. Updated build number.